### PR TITLE
Clarify finite-geometry envelopes and canonical value maps

### LINKED ISSUE #2531: [Tech Debt]: Consolidate duplicate finite-field ProjectivePoint representations
## Problem

The repository has two independent `ProjectivePoint` value contracts for prime-field projective points:

- `src/jacobian/math/finite_geometry/_models.py:40` stores a `PrimeFieldVectorSpace` plus canonical integer residues.
- `src/jacobian/math/finite_fields/values.py:325` stores a `FiniteFieldPresentation`, an `Axis`, and `FiniteFieldElement` coordinates.

Both are normalized projective representatives over a named prime field and coordinate axis, but they are different Pydantic classes with different parent and coordinate representations. The finite-geometry operations (`finite_geometry.projective_point.canonicalize`, equality, and enumeration) therefore cannot hand their result directly to finite-fields operations such as `finite_fields.restrict_scalars` or `finite_fields.linear_map_rank`, which require the other `ProjectivePoint` type.

A direct probe confirms the boundary: serializing a finite-geometry point such as `space={field_order: 5, axis: (x, y)}, coordinates=(1, 2)` and validating it as the finite-fields `ProjectivePoint` raises validation errors because `presentation` and typed field elements are required. There is no domain-owned conversion or shared canonical value.

## Impact

Mathematically equivalent projective points cannot be composed across the finite-geometry and finite-fields catalogs without caller-side reconstruction. That creates duplicate ownership of a canonical value, makes producer-consumer composition depend on private model knowledge, and allows normalization/parent-validation rules to drift.

## Suggested direction

Choose one domain-owned finite-field projective-point representation and make both operation families consume and return it, or add one explicit, tested conversion at the domain boundary. Preserve the existing field/axis and work bounds; this is an ownership and interoperability repair, not a request to merge unrelated operation semantics.

## Evidence

- `finite_geometry/_models.py`: `PrimeFieldVectorSpace` and integer-residue `ProjectivePoint`.
- `finite_fields/values.py`: `FiniteFieldPresentation`, `Axis`, `FiniteFieldElement`, and typed `ProjectivePoint`.
- `finite_geometry/_tools.py`: canonicalization, equality, and enumeration publish the local value.
- `finite_fields/_models.py`: finite-field operations require the separate value.
- Runtime probe: `GeometryPoint(...).model_dump()` is rejected by `FieldPoint.model_validate(...)` with five validation errors.

### PR DESCRIPTION
**Pull request by morluto**
Fixes #2531.
Refs #2581 and #2583.

## Problem

Finite-geometry enumeration enforced a hidden literal cap, Grassmannian counts crossed the JSON boundary as host integers, and projective points had no explicit map into the finite-field value family. That left caller-visible schema guidance incomplete and required consumers to reconstruct equivalent values themselves.

## Solution

Publish the named projective-space enumeration envelope in the request schema and derive validation from the same constant. Encode exact Grassmannian counts as canonical decimal integers so their public representation remains transportable. Add an explicit, checked embedding from prime-field projective points to a caller-selected finite-field presentation and axis; the target parent change is intentional rather than implicit.

## Testing

- `make test-focused LANE=math TESTS="tests/math/finite_geometry tests/math/finite_fields"` — 56 passed
- `make test-focused LANE=catalog TESTS=tests/catalog` — 41 passed
- `make test-focused LANE=dispatch TESTS=tests/dispatch` — 45 passed
- `make test-focused LANE=integration TESTS=tests/integration/catalog/test_builtin_examples.py` — 611 passed
- `git diff --check` — passed

## Public contract impact

`GrassmannianCountResult.count` is now a canonical decimal integer string. Projective-space enumeration publishes its actual `q**n` envelope. Native callers gain `embed_projective_point_in_finite_field`, which requires an explicit compatible target presentation and axis.

## Operation boundary ownership

- Changed stage: request parsing and bounds, result construction, and transport projection
- Request-bounds owner and controlling quantities: finite-geometry projective-space enumeration; `q**len(axis)` is bounded by the named vector envelope
- Backend or kernel path: unchanged exact finite-field arithmetic
- Result construction: Gaussian-binomial counts use canonical decimal encoding
- Independent-result verifier: existing Grassmannian replay remains bounded by the request domain
- Native/MCP parity: the same typed finite-geometry values and admission apply before transport; the new native map has an explicit target parent
- Serialized-result and round-trip evidence: finite-geometry owner and dispatch coverage include the canonical count representation

## Canonical value audit

- [x] Existing canonical values searched
- [x] Intentional finite-field embedding documented
- [x] Producer→consumer composition tested

## Checklist

- [ ] `make check` passes (not run; focused owner, catalog, dispatch, and integration checks passed)
- [x] Runtime-bound change names its request-bounds owner
- [x] Public contract change includes schema-to-run and serialized-boundary coverage

### CODE REVIEW

File: src/jacobian/math/finite_geometry/_models.py [hunk 1]
```diff
@@ -224,13 +227,47 @@ def require_valid(self) -> Self:
...
+    ``q`` is the prime field order and ``n`` is the length of its ordered
+    coordinate axis.
+    """
+
+    model_config = ConfigDict(
+        json_schema_extra={
+            "description": (
+                "One finite projective space whose complete point list fits the "
+                "enumeration envelope. It admits exactly q**n <= "
+                f"{MAX_PROJECTIVE_SPACE_ENUMERATION_VECTORS}, where q is the "
```
[the remark this answers is not in the document]
Comment (morluto) [reply src/jacobian/math/finite_geometry/_models.py]: Fixed in 37d9d7432: results now use compact bare coordinate tuples under a single top-level `space` (reviewer case drops 10.69 MiB → 2.13 MiB), plus an admission-time serialized-result bound derived from the Gaussian-binomial point count and label lengths with typed rejection `finite_geometry.projective_enumeration_result_too_large` before enumeration.

File: src/jacobian/math/finite_geometry/_models.py [hunk 2]
```diff
@@ -397,8 +498,16 @@ def replay(self) -> Self:
...
+
+    Points return as bare canonical coordinate tuples relative to the
+    declared parent ``space``, mirroring the domain's subspace values: the
+    parent appears once, so the reply stays compact for the whole envelope
+    instead of repeating space metadata per point.
+    """
+
     space: PrimeFieldVectorSpace
-    points: tuple[ProjectivePoint, ...]
+    points: tuple[tuple[int, ...], ...]
```
[the remark this answers is not in the document]
Comment (morluto) [reply src/jacobian/math/finite_geometry/_models.py]: Fixed in 9e025fe24: enumeration now returns the domain-owned `ProjectivePointSequence` (space once + compact coordinate tuples on the wire; same transport footprint), whose native `.points` yields typed parent-bound `ProjectivePoint` items that compose directly into `embed_projective_point_in_finite_field` with no manual reconstruction. Replay completeness/residue/uniqueness validation moved into the value itself; the maximal PG(15,F₂) wire-bound regression still holds.

File: src/jacobian/math/finite_geometry/__init__.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
-"""Finite geometry operations."""
+"""Finite geometry operations and explicit canonical-value maps."""
 
-__all__: list[str] = []
+from jacobian.math.finite_geometry.conversions import (
+    embed_projective_point_in_finite_field,
+)
+
+__all__ = ["embed_projective_point_in_finite_field"]
```
[the remark this answers is not in the document]
Comment (morluto) [reply src/jacobian/math/finite_geometry/__init__.py]: Fixed in 9e025fe24: `jacobian.math.finite_geometry` now exports `PrimeFieldVectorSpace`, `ProjectivePoint`, `ProjectivePointSequence`, and the public `projective_point(space, vector)` constructor alongside `embed_projective_point_in_finite_field` under an explicit `__all__`, following the finite-fields domain pattern — native callers construct inputs and compose producer→consumer entirely through documented public API.

File: src/jacobian/math/finite_geometry/values.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+                "point_sequence_points_not_unique",
+                "projective points must be unique",
+            )
+        return self
+
+    def __len__(self) -> int:
+        return len(self.coordinates)
+
+    @property
+    def points(self) -> Iterator[ProjectivePoint]:
```
[the remark this answers is not in the document]
Comment (morluto) [reply src/jacobian/math/finite_geometry/values.py]: Fixed in 43e7f62ec: `ProjectivePointSequence.__iter__` yields typed parent-bound `ProjectivePoint` items, so `for point in result.sequence` composes directly into `embed_projective_point_in_finite_field`; verified at runtime that `list(sequence)` returns ProjectivePoint values.

File: src/jacobian/math/finite_geometry/_models.py [hunk 5]
```diff
@@ -224,16 +181,87 @@ def require_valid(self) -> Self:
...
+        """
+
+        q = self.space.field_order
+        n = len(self.space.axis)
+        digit_width = len(str(q - 1))
+        point_count = (q**n - 1) // (q - 1)
+        per_point_bytes = 2 + n * digit_width + (n - 1) + 1
+        predicted = (
+            _PROJECTIVE_ENUMERATION_ENVELOPE_BYTES
+            + sum(len(encode_strict_json(label)) for label in self.space.axis)
```
[the remark this answers is not in the document]
Comment (morluto) [reply src/jacobian/math/finite_geometry/_models.py]: Fixed in 7fc2ec399: the serialized-result estimate now measures NFC-normalized axis-label encodings, matching how canonical JSON normalizes strings before transport. The U+0344-expansion case (≈6 MiB raw → ≈10.4 MiB normalized) is rejected by typed admission before enumeration; regression test covers it.
